### PR TITLE
feat: project pipeline

### PR DESCRIPTION
**Pull request by hujiahao-hjh**
## What this PR does / why we need it:
Project pipeline

## I have checked the following points:
- [x] I18n is finished and updated by cli
- [x] Form fields validation is added and length is limited
- [x] Display normally on small screen
- [x] Display normally when some data is empty or null
- [x] Display normally in english mode

## Does this PR introduce a user interface change?

✅ Yes(screenshot is required)
image

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |  New project level pipeline. |
| 🇨🇳 中文    | 新增项目级流水线。 |

## Does this PR need be patched to older version?
❎ No

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/config-page/components/table/render-types.tsx [hunk 1]
```diff
@@ -632,7 +635,11 @@ const getTableOperation = (val: any, record: any, extra: any) => {
             onCancel={(e: any) => e && e.stopPropagation()}
             zIndex={1100}
           >
-            <span className="table-operations-btn px-4 py-1 block" onClick={(e: any) => e.stopPropagation()}>
+            <span
+              className="table-operations-btn px-4 py-1 block flex-h-center"
+              onClick={(e: any) => e.stopPropagation()}
```
Comment (daskyrk) [inline shell/app/config-page/components/table/render-types.tsx:640]: useless any

File: shell/app/modules/project/pages/pipelines/components/logs.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            },
+            props: {
+              styleNames: 'h-full',
+              wrapperClassName: 'flex-1',
+            },
+          },
+        }}
+      />
+      <Drawer onClose={onClose} visible={visible} width="80%" destroyOnClose closable={false}>
+        <PipelineRunDetail deployAuth={{ hasAuth: false }} isMobileInit={false} />
```
Comment (daskyrk) [inline shell/app/modules/project/pages/pipelines/components/logs.tsx:56]: hasAuth is hardcode false ?

File: shell/app/config-page/components/table/render-types.tsx (hunk 1 above)
Comment (hujiahao-hjh) [reply shell/app/config-page/components/table/render-types.tsx:640]: resolve on https://github.com/erda-project/erda-ui/pull/2610;

**Pull request merged**